packages/react-core/src/components/Select/Select.tsx

```diff
@@ -159,6 +159,8 @@ export interface SelectProps
   isInputValuePersisted?: boolean;
   /** @beta Flag for retaining filter results on blur from keyboard-entered typeahead text */
   isInputFilterPersisted?: boolean;
+  /** Flag indicating that the state of Select such as typeahead input should reset upon selection */
```

Comment:
is it the state or the value of the input? as is, this prop sounds very destructive 😳

Reply:
I'll update the description to remove the state mention as its internal.

Will also look into the description popping up.